datacube/api/core.py

```diff
@@ -350,14 +348,32 @@ def group_datasets(datasets, group_by):
 
         .. seealso:: :meth:`find_datasets`, :meth:`load_data`, :meth:`query_group_by`
         """
+        if isinstance(group_by, str):
+            group_by = query_group_by(group_by=group_by)
+
         dimension, group_func, units, sort_key = group_by
```

Comment:
Then `Datacube.load` can also just pass the `group_by` parameter directly to `group_datasets`?

Reply:
should be able to yes

Reply:
Sorry no, since it might not be present in the `.load` call, also `query_group_by` seems to take all other query parameters just in case future groupings need more args?